### PR TITLE
Fix #3592

### LINKED ISSUE #3592: 'BlenderVerifier' object has no attribute 'callback'
Golem 0.18.1

```
2018-11-23 01:23:29 INFO     apps.blender.verification           Running verification of subtask '3cae51d8-eeb8-11e8-9b19-1f8e0cdf9006'
2018-11-23 01:23:29 CRITICAL twisted                             Unhandled Error
Traceback (most recent call last):
  File "/data/golem/golem_git/golemapp.py", line 184, in start
    node.start()
  File "/data/golem/golem_git/golem/node.py", line 133, in start
    self._reactor.run()
  File "/data/golem/golem_git/venv/lib/python3.6/site-packages/twisted/internet/base.py", line 1243, in run
    self.mainLoop()
  File "/data/golem/golem_git/venv/lib/python3.6/site-packages/twisted/internet/base.py", line 1252, in mainLoop
    self.runUntilCurrent()
--- <exception caught here> ---
  File "/data/golem/golem_git/venv/lib/python3.6/site-packages/twisted/internet/base.py", line 851, in runUntilCurrent
    f(*a, **kw)
  File "/data/golem/golem_git/apps/blender/verification_queue.py", line 87, in callback
    self._process_queue()
  File "/data/golem/golem_git/apps/blender/verification_queue.py", line 66, in _process_queue
    self._run(entry, verifier_cls)
  File "/data/golem/golem_git/apps/blender/verification_queue.py", line 95, in _run
    result = entry.start(verifier_cls)
  File "/data/golem/golem_git/apps/blender/verification_task.py", line 24, in start
    self.verifier.task_timeout(self.subtask_id)
  File "/data/golem/golem_git/venv/lib/python3.6/site-packages/golem_verificator/verifier.py", line 65, in task_timeout
    self.callback(subtask_id=subtask_id,
builtins.AttributeError: 'BlenderVerifier' object has no attribute 'callback'
```

### PR DESCRIPTION
**Pull request by Elfoniok**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -31,8 +31,8 @@ eth-keys==0.2.0b3
 eth-tester==0.1.0-beta.24
 eth-utils==1.0.3
 ethereum==1.6.1
-Golem-Verificator==1.6.1
 Golem-Messages==2.17.0
+Golem-Verificator==1.6.2
```
Comment (etam) [inline requirements.txt]: why move?

**Comment by etam** [comment]
@Elfoniok In this change https://github.com/golemfactory/golem-verificator/commit/e645b00c69229ffedef083c7f18efb00330bd34d (that you pushed without any review), in `def task_timeout` there is `self._clear_state()` after a `return` statement.

File: apps/blender/verification_task.py [hunk 2]
```diff
@@ -21,8 +21,9 @@ def start(self, verifier_class) -> Optional[Deferred]:
             deferred.callback(self.verifier.verification_completed())
             return deferred
         else:
-            self.verifier.task_timeout(self.subtask_id)
-        return None
+            deferred = Deferred()
+            deferred.callback(self.verifier.task_timeout(self.subtask_id))
+            return deferred
```
Comment (etam) [inline apps/blender/verification_task.py]: https://twistedmatrix.com/documents/18.9.0/api/twisted.internet.defer.html#succeed

Comment (etam) [reply apps/blender/verification_task.py]: Is the return type still `Optional[Deferred]`? It seems it's now just `Deferred`.

File: requirements.txt (hunk 1 above)
Comment (jiivan) [reply requirements.txt]: almost alphabetical order? ;)

**Review by jiivan — APPROVED** [review]
I like @etam s suggestions. LGTM

File: requirements.txt (hunk 1 above)
Comment (etam) [reply requirements.txt]: small steps at a time :)

Comment (Elfoniok) [reply requirements.txt]: Sometimes it happen during merge, actually i like it more as it is sorted in increasing order of characters in `Golem-` deps :P

File: apps/blender/verification_task.py (hunk 2 above)
Comment (Elfoniok) [reply apps/blender/verification_task.py]: good points

**Pull request merged**